eap-xp4-jdk17/image.yaml

```diff
@@ -35,6 +35,10 @@ ports:
     - value: 8443
 modules:
       repositories:
+          - name: openjdk
+            git:
+                  url: https://github.com/jboss-container-images/openjdk
+                  ref: wildfly-container-29.0
```

Comment:
@jfdenise are we using the same `ref` for this repository between JDK 11 & 17 images?

Reply:
@jmesnil , yes, this usage started with EAP 8.0 images, The never released JDK11 image and released JDK17 image use the same tag.